admin_manual/enterprise/external_storage/s3_swift_as_primary_object_store_configuration.rst

```diff
@@ -1,159 +1,167 @@
-=============================================================
-Configuring S3 and OpenStack Swift Objects as Primary Storage
-=============================================================
+=================================
+Configuring S3 as Primary Storage
+=================================
 
-In ownCloud Enterprise edition, you can configure S3 objects as primary 
-storage. This replaces the default ownCloud ``owncloud/data`` directory. You may 
-still need to keep the ``owncloud/data`` directory for these reasons:
+In ownCloud Enterprise edition administrators can configure Amazon S3 objects as the primary storage.
+Doing this replaces the default ownCloud ``owncloud/data`` directory.
+
+You may still need to keep the ``owncloud/data`` directory for these reasons:
 
 * The ownCloud log file is saved in the data directory
 * Legacy apps may not support using anything but the ``owncloud/data`` directory
 
-You can move your logfile by changing its location in ``config.php``. You may still need 
-``owncloud/data`` for backwards compatibility with some apps.
+Administrators can move their ownCloud log file, by changing its location in ``config.php``.
+However, even when this is done, ``owncloud/data`` may still be required for backward compatibility with some apps.
+
+
+`The S3 Object Storage app`_ (``files_primary_s3``) is the preferred way to provide S3 storage support,
+However, `the Object Storage Support app`_ (``objectstore``) is still available.
+
+.. note:: OpenStack Swift has been deprecated.
+
+When using ``files_primary_s3``, the Amazon S3 bucket needs to be created manually according to the developer documentation, and versioning needs to be enabled.
+
+.. note:: ownCloud GmbH provides consulting for migrations from ``objectstore`` to ``files_primary_s3``.
+
+.. note:: The source of the S3 Object Storage app `is available on gitHub`_.
```

Comment:
sure? no code integrity ... ?
isnt this to be release on the marketplace soon? I remember that @pvince and @deepdiver said something about having it on the MP.
I would at least add a comment like availabe on the market place soon or so

Reply:
I did, near the top of the file, as "_The S3 Object Storage app_".

Reply:
ok, got it
is it necessary to link to the source on github, what is the intention behind?
for me this is confusing and looks like pls review the code 😄 ?
all free of charge apps are visible and available on github anyways